rs/nervous_system/integration_tests/tests/advance_sns_target_version.rs

```diff
@@ -195,7 +197,7 @@ async fn test_get_upgrade_journal() {
 
     await_with_timeout(
         &pocket_ic,
-        UPGRADE_STEPS_INTERVAL_REFRESH_BACKOFF_SECONDS,
+        0..10,
```

Comment:
Can we assign this `10` to a constant and give it a name, e.g., `EXPECTED_REFRESH_SECONDS: u64 = 10`?

Reply:
Done!